api/app/Models/Ads/OzonAd.php

```diff
@@ -0,0 +1,128 @@
+        private ?string $ozonInstock4 = null;
+        private ?string $ozonWarehouseName5 = null;
+        private ?string $ozonInstock5 = null;
+        private ?string $ozonWarehouseName6 = null;
+        private ?string $ozonInstock6 = null;
+        private ?string $ozonWarehouseName7 = null;
+        private ?string $ozonInstock7 = null;
+        private ?string $ozonWarehouseName8 = null;
+        private ?string $ozonInstock8 = null;
+        private ?string $ozonWarehouseName9 = null;
+        private ?string $ozonInstock9 = null;
+        private ?string $ozonWarehouseName10 = null;
+        private ?string $ozonInstock10 = null;
+        
+        public function __construct(array $row, TableHeader $propertyColumns)
+        {
+            parent::__construct($row, $propertyColumns);
+            
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonOfferId');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonPrice');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonOldPrice');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonPremiumPrice');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonWarehouseName');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonInstock');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonWarehouseName2');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonInstock2');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonWarehouseName3');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonInstock3');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonWarehouseName4');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonInstock4');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonWarehouseName5');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonInstock5');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonWarehouseName6');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonInstock6');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonWarehouseName7');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonInstock7');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonWarehouseName8');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonInstock8');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonWarehouseName9');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonInstock9');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonWarehouseName10');
+            $this->checkAndSetProp($row, $propertyColumns, 'ozonInstock10');
+        }
+    
+        /**
+         * @param array       $row
+         * @param TableHeader $propertyColumns
+         * @param string      $prop
+         */
+        private function checkAndSetProp(array $row, TableHeader $propertyColumns, string $prop): void
```

Comment:
You can safely write checkAndSetProperty. No one will blame you for readability at cost of 5 symbols